### PR TITLE
Bump github.com/containerd/containerd from 1.2.9 to 1.3.0

### PR DESCRIPTION
**Pull request by saschagrunert**
- [Release notes](https://github.com/containerd/containerd/releases)
- [Changelog](https://github.com/containerd/containerd/blob/master/RELEASES.md)
- [Commits](https://github.com/containerd/containerd/compare/v1.2.9...v1.3.0)

### CODE REVIEW

File: internal/oci/runtime_vm.go [hunk 1]
```diff
@@ -177,7 +177,9 @@ func (r *runtimeVM) startRuntimeDaemon(c *Container) error {
 		r.ctx,
 		newRuntimePath,
 		"",
+		"",
 		c.BundlePath(),
+		nil,
```
Comment (saschagrunert) [inline internal/oci/runtime_vm.go]: /cc @sboeuf

Comment (TomSweeneyRedHat) [reply internal/oci/runtime_vm.go]: I"m a bit confused by this change and your comment (or was it the bot's comment?).

Comment (saschagrunert) [reply internal/oci/runtime_vm.go]: Hm no, I was just wondering if the containerd API changes somehow have any (maybe bad) influence on the runtime_vm implementation. I would not suggest it, but added a pointer to it :)

**Comment by haircommander** [comment]
/test e2e_features_fedora
this is a known flake, but I need a vm to debug

**Comment by haircommander** [comment]
hey @saschagrunert can you rebase to pick up CI fixes :)

**Comment by saschagrunert** [comment]
Rebased on top of the latest master

**Comment by saschagrunert** [comment]
Rebased on top of the latest master, PTAL.

**Comment by TomSweeneyRedHat** [comment]
LGTM and happy green test buttons

**Pull request merged**